.env-dist

```diff
@@ -4,3 +4,6 @@ CONTENT_ROOT=../content/files
 
 # See documentation in docs/envvars.md for more information about this
 #BUILD_FLAW_LEVELS=broken_links:error, macros:ignore
+
+# Use following to put the local dev server in developer.mozilla.org's time zone
+#TZ=America/Chicago
```

Comment:
Currently, all SSR pages are statically generated as part of our daily/nightly build. This means they use GitHub action runner's timezone (UTC), not America/Chicago.

TIL You can influence the timezone by setting that environment variable, but I'm not sure the `.env-dist` is the right place to document this, as it remains an edge-case. :/

```suggestion
```